mlrun/projects/pipelines.py

```diff
@@ -723,7 +723,7 @@ def run(
                 name=runner_name,
                 project=project.name,
                 kind="job",
-                image=mlrun.mlconf.default_base_image,
+                image=mlrun.mlconf.default_workflow_runner_image,
```

Comment:
why?

Reply:
To let the user to set his custom image for running workflows (remotely of course) without changing the `default_base_image` of mlrun

Reply:
I see. if it has nothing with slack pr -> move to a different PR
try to avoid mixing-up such changes, it will allow maintainers to revert prs without undo-ing good fixes

Reply:
also, the configuration you have changed here is cross-mlrun, how will I be able to set a custom image without changing other's defaults?

Reply:
np, removed.